crates/holochain_types/src/dht_op.rs

```diff
@@ -382,7 +382,7 @@ impl DhtOp {
         }
     }
 
-    /// Convert a [DhtOp] to a [DhtOpLite] and basis
```

Comment:
Inaccurate comment, the basis is actually inside the DhtOpLite and not returned in a tuple or something like that.